### ISSUE TITLE
Add a 'Downloaded' Category

### ISSUE THREAD

**Issue by VektorOO7** [mihonapp/mihon#1358]
### Describe your suggested feature

Add the option to include 'downloaded' category, where you can find all the mangas in any category that are downloaded. The category should not be deletable, but it can be turned on or off from settings.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Smol-Ame** [comment]
... but what about just Filtering by Downloaded, or enabled "Downloaded only"?

![Screenshot_20241022_143913_Mihon.png](https://github.com/user-attachments/assets/d59a5003-6c7a-4a2e-8bc2-de066e7163f3)

![Screenshot_20241022_143928_Mihon.png](https://github.com/user-attachments/assets/90692183-51b9-4432-9d3f-47eae1beb78f)

**Comment by VektorOO7** [comment]
> ... but what about just Filtering by Downloaded, or enabled "Downloaded only"?
> 
> ![Screenshot_20241022_143913_Mihon.png](https://github.com/user-attachments/assets/d59a5003-6c7a-4a2e-8bc2-de066e7163f3)
> 
> ![Screenshot_20241022_143928_Mihon.png](https://github.com/user-attachments/assets/90692183-51b9-4432-9d3f-47eae1beb78f)
> 
> 

Yeah, but I just thought it would be more convenient to add it as a category, similar to how in Kotatsu you can check all of the downloaded manga.

**Comment by BrutuZ** [comment]
You can have entries in multiple categories, just have one category with all your stuff in addition to your usual organization ones, enable the Downloaded filter and voila, you have this magic functionality you wished for
